lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -104,14 +104,14 @@ typedef dynamic Dispatcher(dynamic action);
 ///     Store store1 = new Store<CounterState>(counterStateReducer, initialState: new CounterState(count: 0));
 ///     Store store2 = new Store<BigCounterState>(bigCounterStateReducer, initialState: new BigCounterState(bigCount: 100));
```

Comment:
#supernit while we're here 🤷‍♂ 
```suggestion
///     Store store1 = Store<CounterState>(counterStateReducer, initialState: new CounterState(count: 0));
///     Store store2 = Store<BigCounterState>(bigCounterStateReducer, initialState: new BigCounterState(bigCount: 100));
```